app/src/main/java/comcmput301f16t01/github/carrier/Users/NewUserActivity.java

```diff
@@ -31,34 +31,36 @@ public void onBackPressed() {
         this.finish();
     }
 
+
     /**
-     * @param v The button (view) the user presses that calls this method.
+     * This is the function that will take us to the next screen.
+     * We will bundle all info entered, check validity and then send it to next activity
+     * @param v the view needed for the function
      */
-    public void submitNewUser(View v) {
+
+    public void nextVehicle(View v) {
```

Comment:
I am not sure I like this name. Could we try something more descriptive? I get what you're trying to do here but I think that nextVehicle is kind of confusing...I'm sorry I can't think of a better name at the mo...I will report back...